src/i18n/langResources/lang.de.ts

```diff
@@ -27,6 +27,7 @@ export const langDe: ILangResource = {
   tsOfficialPresetMessage: 'Offizielles TS Icon wird',
   jsonOfficialPresetMessage: 'Offizielles JSON Icon wird',
   hideFoldersPresetMessage: 'Ordner Icons werden',
+  foldersAllDefaultIconPresetMessage: 'Ordner Icons Auflegemaske werden',
```

Comment:
wut, completly no. `Out-of-Box` is not even German 😄 Sounds like Google Translate doesn't translate unknown vocabularies. As I proposed before: `Standardmäßige Ordner-Icons aktivieren/deaktivieren`